### ISSUE TITLE
[Accessibility] The focus cannot  turn clockwise and turn counter clockwise successfully by using Tab & Shift + Tab to navigate in MenuStrip

### ISSUE THREAD

**Issue by Jenna-Zheng** [dotnet/winforms#3835]
* .NET Core Version: 
.NET SDK 5.0.100-rc.1.20453.7

* Have you experienced this same bug with .NET Framework?: 
Yes

* More Info
The BindingNavigator/MenuStrip/ToolStrip also reproduce this issue.

**Problem description:**
Steps2: The focus Stay in the first Item of ToolStrip by using **Tab & Shift + Tab** to navigate.
Steps7: The focus both turn clockwise by using **Tab & Shift + Tab** to navigate now, 
![TestResult_focus](https://user-images.githubusercontent.com/64949616/92193203-4c422900-ee9a-11ea-9e4b-f539c9c669ae.gif)

**Expected behavior:**
Steps2:  The focus should turn clockwise and turn counter clockwise successfully.
Steps7: The focus should turn counter clockwise by using **Shift + Tab** to navigate, 

**Minimal repro:**
1. Extract "[WindowsFormsApp2.zip](https://github.com/dotnet/winforms/files/5172264/WindowsFormsApp2.zip)" and open it in VS(the **ToolStrip's TabStop** property value is **true** in this project)
2. Build and run the project.
3. Navigate the **ToolStrip** by using **Tab & Shift + Tab** and observe.
4. Stop the project and set **ToolStrip's** **TabStop** property value to **false**.
5. Build and run the project again.
6. Switch focus to **TextBox** item in **ToolStrip**.
7. Using **Tab & Shift + Tab** to navigate **ToolStrip's** items and observe.

**Comment by Olina-Zhang** [comment]
This is by design, because setting ToolStrip's TabStop property value to false. Here is reference:  true if the user can give the focus to the control using the TAB key; otherwise, false. from MSDN: https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.control.tabstop?view=netcore-3.1

**Comment by merriemcgaw** [comment]
@Olina-Zhang can we go ahead and make sure the accessibility sample app is set up to demonstrate that we can set it to respond to TAB key?

**Comment by Olina-Zhang** [comment]
A little corrections: For step2, when TabStop = true, cannot use Tab & Shift + Tab to navigate toolStripItems in menuStrip/ToolStrip --- it is by design. And for step7, when TabStop = false, can use Tab & Shift + Tab to navigate, but both turn clockwise, Shift + Tab should turn counter clockwise --- It is a issue in here.
@merriemcgaw if setting TabStop = true in menuStrip/ToolStrip, we can just use Tab to go through toolstripItems inside these controls, and fall into them, cannot jump out and tab to other control in same form designer.

**Comment by Lydia-Shi** [comment]
Verified this issue with .Net 6.0.100-rc.2.21466.17, this issue is fixed. When setting TabStop=True, focus stay in the first Item by using Tab and focus stay in last item by using Shift + Tab to navigate. When setting TabStrop=false, focus turn counter clockwise by using Shift + Tab to navigate, results as below:
![toolStrip](https://user-images.githubusercontent.com/54019172/133713083-6d6fae94-4406-47c6-b46d-c58065e4e858.gif)

**Comment by Lydia-Shi** [comment]
Verified this issue on .NET 6.0 RC2 test pass build: 6.0.100-rc.2.21473.26, it was fixed.
